src/stories/Library/disclosure/disclosure.scss

```diff
@@ -0,0 +1,32 @@
+.disclosure {
+  width: auto;
+  border: solid 1px $c-global-tertiary-1;
+
+  @include breakpoint-m {
+    margin: 0 133px;
+  }
+
+  &__headline {
+    display: flex;
+    align-items: center;
+    justify-content: baseline;
+    cursor: pointer;
+  }
+
+  &__icon {
+    height: 40px;
+    width: 40px;
+    border-radius: 20px;
+    display: flex;
+    justify-content: center;
+
+    & > img {
+      width: 24px;
+      filter: invert(1);
```

Comment:
If I understand correct, this is to change the color of a white icon?
So i'm not sure it is the best way, to have a consistant color across browsers. Also it is not obvious what color you end up with.
Since it is a svg that is used, we might be able to create a more flexible solution.

Reply:
Mmm yeah this was very much a solution I found because other things were not working. The problem is that the SVG is set as a source for an <img> tag. Which basically removes all the great features SVG files have. I tested if it works well in all the browsers we need, and it seems like it's behaving fine and consistent.